### PR TITLE
Filter only commit first sentence for changelog exceptions

### PR DESCRIPTION
**Pull request by guizmaii**
Related to https://github.com/conduktor/conduktor-desktop/pull/243

### CODE REVIEW

File: .github/scripts/release.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ NAME="$VERSION ($DATE)"
 
 CURRENT_TAG=$(git describe --abbrev=0 --tags)
 PREVIOUS_TAG=$(git describe --abbrev=0 --tags $(git describe --abbrev=0)^)
-CHANGELOG=$(git log --no-merges --pretty=%s "$PREVIOUS_TAG".."$CURRENT_TAG" --invert-grep --grep="Bomb" --grep "skip ci" --grep "skip changelog" -i | sed -e 's/^/- /' | sed -E 's/\(#[0-9]+\)//')
+CHANGELOG=$(git log --no-merges --pretty=%s "$CURRENT_TAG".. | egrep -v "skip ci|skip changelog" -i | sed -e 's/^/- /' | sed -E 's/\(#[0-9]+\)//')
```
Comment (guizmaii) [inline .github/scripts/release.sh:21]: @sderosiaux @gaspb Do we know why there was a difference between CDK script and this script around the `"$PREVIOUS_TAG".."$CURRENT_TAG"`

Here, it was `"$PREVIOUS_TAG".."$CURRENT_TAG"` while in CDK it was only `"$CURRENT_TAG"..`?

Comment (sderosiaux) [reply .github/scripts/release.sh:21]: on master: we work until the latest, here, we work between tags (not the latest which can have more commits already)

Comment (guizmaii) [reply .github/scripts/release.sh:21]: Ok I change then

Comment (guizmaii) [reply .github/scripts/release.sh]: Changed

**Pull request merged**